main.tex

```diff
@@ -497,21 +499,21 @@ \subsection*{\hypertarget{Combatterminology}{Combat terminology}}
 
 \subsection*{\hypertarget{CombatCards}{Using cards during combat}}
-You may only use \textbf{one spell per combat round}. Other card types can be used as many times as you want to. Ongoing \includegraphics[scale=0.75]{images/ongoing.PNG} and \includegraphics[scale=0.3]{images/unit_activation.PNG} activate effects can be used only when activating a unit and before it attacks. Ongoing effects last until end of combat or if the effect on the card is used up. Instant \includegraphics[scale=0.75]{images/instant.PNG} cards may be played at any time except between rolling the combat die and resolving damage unless otherwise stated. Instant cards which increase a unit's statistics \textbf{last only for the duration of the current unit's activation}.
+You may only use \textbf{one spell per combat round}. Other card types can be used as many times as you want to. Ongoing \includesvg[height=10px]{svgs/ongoing.svg} and \includesvg[height=10px]{svgs/activation.svg} activate effects can be used only when activating a unit and before it attacks. Ongoing effects last until end of combat or if the effect on the card is used up. Instant \includesvg[height=10px]{svgs/instant.svg} cards may be played at any time except between rolling the combat die and resolving damage unless otherwise stated. Instant cards which increase a unit's statistics \textbf{last only for the duration of the current unit's activation}.
 \subsection*{\hypertarget{Timelimit}{Combat time limits}}
 Combat against neutral units has a time limit of one round. If the player cannot win the combat
 before the end of the current combat round, they must either \hyperlink{Endcombat}{retreat} or spend 1 MP from the hero who started the combat in order to play another combat round. Combat against azure units, other players or AI heroes has no time limit.
 \clearpage
 \subsection*{\hypertarget{Walls}{Defending a town with a citadel}}
-When a town with a citadel is attacked, the defender adds the wall and gate obstacles in any order to the middle row of the combat board after placing their units. The gate card is not an obstacle to the defending player. The wall and gate cards can be destroyed by any adjacent \includegraphics[scale=0.25]{images/ground_icon.PNG} or \includegraphics[scale=0.25]{images/flying_icon.PNG} unit's attack. Such attacks are always successful, do not roll the combat die when performing them.\\[6pt]
+When a town with a citadel is attacked, the defender adds the wall and gate obstacles in any order to the middle row of the combat board after placing their units. The gate card is not an obstacle to the defending player. The wall and gate cards can be destroyed by any adjacent \includesvg[height=10px]{svgs/unit_ground.svg} or \includesvg[height=10px]{svgs/unit_flying.svg} unit's attack. Such attacks are always successful, do not roll the combat die when performing them.\\[6pt]
```

Comment:
It's been a while since I've used LaTeX myself, but I wonder if it's possible to use a variable instead of writing `10px` everywhere? Say somebody decides to use 11px or 9px in the future, this change would be a lot less painful.

Reply:
It can be done, thanks for the suggestion.

Reply:
I guess there are a number of hardcoded values like that now. We could help @Heegu-sama identify them and put them in the beginning of the document as variables.

Reply:
Yes, the most common of these is the constant [6pt] when changing to a new paragraph. The default paragraph spacing should be defined as that.